_includes/tutorials/error-handling/kstreams/markup/dev/build-uberjar.adoc

```diff
@@ -0,0 +1,14 @@
+Now that we have data generation working, let's build your application by running:
+
+```
+NOTE You need to run following commands to copy the SNAPSHOT Jars to the correct location since AK 2.8 is not in Maven Central yet
+
+mkdir temp_local_28_repo
+cp <KAFKA-TUTORIALS-REPO-DIRECTORY>/_includes/tutorials/error-handling/kstreams/code/temp_local_28_repo/* temp_local_28_repo
+
+DELETE This section before the relase of this tutorial!!!
+```
```

Comment:
This step is here for reviewers who want to run the tutorial locally.  This step is needed to get the necessary jar files available for the build.  This will not be needed once AK 2.8 jars are available in Maven Central.

These lines need to be deleted for merging.